crates/proof-of-sql/src/proof_primitive/dory/dory_vmv_helper.rs

```diff
@@ -56,7 +56,7 @@ pub(super) fn compute_T_vec_prime(
 
     prover_setup.blitzar_msm(
         &mut blitzar_commits,
-        data_size as u32,
+        u32::try_from(data_size).unwrap_or(u32::MAX),
```

Comment:
rather than `unwrap_or`, I think this one should be `.expect("the size of F should fit in u32")`.